MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,12 +20,16 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedLoadError: Observer<String?>?
```

Comment:
Suggestion:
`onFeedLoadError` can be named as `onErrorStateChange` to be consistent with `onLoadingStateChange`
```suggestion
	var onErrorStateChange: Observer<String?>?
```

Reply:
@danillahtin I have updated the name of the variable